gluoncv/utils/metrics/tracking.py

```diff
@@ -0,0 +1,233 @@
+import numpy as np
+from colorama import Style, Fore
+
+def overlap_ratio(rect1, rect2):
```

Comment:
For each metric, please add more description, such as what is the metric, which paper proposes this metric, what is the input, what is the data type, what is the default value, what is the output, etc. Because usually these metrics are shared by many tasks or datasets. Keep a good documentation will help others contribute to GluonCV.